.devcontainer/devcontainer.json

```diff
@@ -1,28 +1,23 @@
-{
-	"name": "Dart (Community)",
-	"build": {
-		"dockerfile": "Dockerfile",
-		// Update VARIANT to pick a Dart version
-		"args": { "VARIANT": "latest" }
-	},
-
-	// Configure tool-specific properties.
-	"customizations": {
-		// Configure properties specific to VS Code.
-		"vscode": {
-			// Add the IDs of extensions you want installed when the container is created.
-			"extensions": [
-				"dart-code.dart-code"
-			]
-		}
-	},
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
 
-	// Use 'forwardPorts' to make a list of ports inside the container available locally.
-	// "forwardPorts": [],
+{
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
 
-	// Use 'postCreateCommand' to run commands after the container is created.
-	// "postCreateCommand": "uname -a",
+  "updateContentCommand": "tool/gh_codespaces/run_setup.sh",
 
-	// Set `remoteUser` to `root` to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-	"remoteUser": "vscode"
+  "customizations": {
+    "codespaces": {
+      "openFiles": [
+        "README.md",
```

Comment:
I think these are great to have, but another that might be nice is `example/example.dart` so that first-time openers would immediately see a file they can run.

Reply:
I added a file with an example and put it first in the queue for opening.

From what I understand, `README.md` is being completely redesigned, so I suggested adding an badge to <https://github.com/intel/rohd/pull/221#discussion_r1098544124>